src/palakSJun20/ValidatePhoneNumber.java

```diff
@@ -0,0 +1,97 @@
+/*Assignment 24 :- 5th Jul'2020 
+Validate given phone number.
+
++91 9765463742 --> valid
+09765463742 --> valid
+976 546 3742 --> valid
+9765463742 --> valid
+
+9765463A42 --> Invalid
+976546342 --> Invalid
+6976546342 --> Invalid
+
+Note : phone number is given in String format.*/
+package palakSJun20;
+
+public class ValidatePhoneNumber {
+
+	void validatePhoneNumber(String pNumber) {
+	
+		// 1st Case - +91 9765463742 have to check for + sign
+		boolean flag = false;
+		if (pNumber.startsWith("+") && pNumber.length() == 14) {
+
+			for (int index = 1; index < pNumber.length(); index++) {
+				if ((Character.isDigit((pNumber.charAt(index))) || Character.isSpaceChar((pNumber.charAt(index))))
```

Comment:
please check the special character logic .if user send the number like +91 976@463742.it will go print number is valid and invalid.

Reply:
Condition updated . 
Thanks for thoroughly validation.